src/main/resources/application.yaml

```diff
@@ -308,9 +308,11 @@ notifications:
   mediationUnsuccessfulClaimantLRTemplate: "86f00ab5-152d-47f3-a1f4-33b6ea7e494d"
   mediationUnsuccessfulDefendantLIPTemplate: "14543b94-cab2-43dc-b061-8a9671bff178"
   mediationUnsuccessfulDefendantLIPBilingualTemplate: "f8841cc4-0357-446d-bbb6-a8c0ff6b53c4"
+  notifyApplicantForHwFMoreInformationNeeded: "f8841cc4-0357-446d-bbb6-a8c0ff6b53c4"
```

Comment:
DO NOT MERGE THIS.
This exists in the commons version. I had to add this so it doesn't fail on the pipeline.